src/mkdocs_gallery/gen_single.py

```diff
@@ -728,10 +728,11 @@ def _get_code_output(is_last_expr, script: GalleryScript, logging_tee, images_md
     return code_output
 
 
-def _reset_cwd_syspath(cwd, sys_path):
+def _reset_cwd_syspath(cwd, new_path):
     """Reset cwd and sys.path."""
```

Comment:
Please replace this with `"""Reset current working directory to `cwd` and remove `path_to_remove` from `sys.path`."""`